Software/scan_key/on_off.py

```diff
@@ -52,3 +52,10 @@ def blinkKey():
     ledOn(blue)
     t.sleep(.1)
     ledOff(blue)
+
+def blinkKey2():
```

Comment:
I'll leave this portion of the code as is until the two members assigned to this task finish it or at least start it.